hc/api/tests/test_badge.py

```diff
@@ -12,13 +12,17 @@ def setUp(self):
         self.check = Check.objects.create(user=self.alice, tags="foo bar")
 
     def test_it_rejects_bad_signature(self):
-        r = self.client.get("/badge/%s/12345678/foo.svg" % self.alice.username)
-        ### Assert the expected response status code
+        resp = self.client.get("/badge/%s/12345678/foo.svg" % self.alice.username)
```

Comment:
While the string formatting statement is valid, I recommend using `format`.

**Resource**: https://goo.gl/NCHBiE